[2024-04-08T01:46:04.589+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: analytics_pipeline.wait_for_archive manual__2024-04-07T20:15:32.965247+00:00 [queued]>
[2024-04-08T01:46:04.595+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: analytics_pipeline.wait_for_archive manual__2024-04-07T20:15:32.965247+00:00 [queued]>
[2024-04-08T01:46:04.595+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-08T01:46:04.609+0530] {taskinstance.py:2191} INFO - Executing <Task(FileSensor): wait_for_archive> on 2024-04-07 20:15:32.965247+00:00
[2024-04-08T01:46:04.616+0530] {standard_task_runner.py:60} INFO - Started process 79827 to run task
[2024-04-08T01:46:04.619+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'analytics_pipeline', 'wait_for_archive', 'manual__2024-04-07T20:15:32.965247+00:00', '--job-id', '293', '--raw', '--subdir', 'DAGS_FOLDER/analytics_pipeline.py', '--cfg-path', '/tmp/tmpv4o9ka1x']
[2024-04-08T01:46:04.620+0530] {standard_task_runner.py:88} INFO - Job 293: Subtask wait_for_archive
[2024-04-08T01:46:04.654+0530] {task_command.py:423} INFO - Running <TaskInstance: analytics_pipeline.wait_for_archive manual__2024-04-07T20:15:32.965247+00:00 [running]> on host LAPTOP-HO799B2R.
[2024-04-08T01:46:04.706+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='analytics_pipeline' AIRFLOW_CTX_TASK_ID='wait_for_archive' AIRFLOW_CTX_EXECUTION_DATE='2024-04-07T20:15:32.965247+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-07T20:15:32.965247+00:00'
[2024-04-08T01:46:04.713+0530] {base.py:83} INFO - Using connection ID 'fs_default' for task execution.
[2024-04-08T01:46:04.713+0530] {filesystem.py:66} INFO - Poking for file /home/lucky/Assign2/archive.zip
[2024-04-08T01:46:04.713+0530] {filesystem.py:71} INFO - Found File /home/lucky/Assign2/archive.zip last modified: 20240408013910
[2024-04-08T01:46:04.713+0530] {base.py:295} INFO - Success criteria met. Exiting.
[2024-04-08T01:46:04.717+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=analytics_pipeline, task_id=wait_for_archive, execution_date=20240407T201532, start_date=20240407T201604, end_date=20240407T201604
[2024-04-08T01:46:04.752+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-08T01:46:04.785+0530] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
